helmfile/values/fluentd-user.yaml.gotmpl

```diff
@@ -120,6 +133,44 @@ extraConfigMaps:
     @include /etc/fluent/extra-config.d/*.conf
 
   output.conf: |-
+    <match authlog.**>
+       @id elasticsearch_authlog
+       @type elasticsearch
+       @log_level "#{ENV['OUTPUT_LOG_LEVEL']}"
+       include_tag_key true
```

Comment:
Seems kind of unessecary to include the tag as a field.

Reply:
Yes, removed it